packages/utils/eslint-config-custom/front.js

```diff
@@ -49,6 +49,12 @@ module.exports = {
             message:
               "'Stack' has been deprecated. Please import 'Flex' from '@strapi/design-system' instead.",
           },
+          {
+            name: '@strapi/helper-plugin',
+            importNames: ['request'],
+            message:
+              "'request' has been deprecated. Please import 'useFetchClient' from '@strapi/helper-plugin' instead.",
```

Comment:
Should we also add the deprecation message to the `request` call directly in case plugin developer rely on it?

Reply:
Yes, i'll add the `once` function here, so it doesn't spam the console 🙃